gpt_index/docstore/__init__.py

```diff
@@ -0,0 +1,10 @@
+from gpt_index.docstore.types import DocumentStore
```

Comment:
i'd name this BaseDocumentStore, and keep DocumentStore for backwards compat with SimpleDocumentStore

Reply:
Revised.